frontend/stylelint.config.cjs

```diff
@@ -0,0 +1,11 @@
+// Using CommonJS (.cjs) to avoid the need for --config-type=module (only in Stylelint v16+).
+// We're on Stylelint v15 for compatibility with stylelint-scss, since v5.3.x breaks on Node 20.
+// This avoids runtime issues and works better with Yarn v1, CI, and cross-platform setups.
```

Comment:
stylelint maintainer here

I am wondering what we could do to help you migrating to stylelint version 16 smoothly.